### ISSUE TITLE
Kong 2.1.4 to 2.3.3 migration certificates resources breaking behavior

### ISSUE THREAD

**Issue by jeremyjpj0916** [Kong/kong#6951]
### Summary

Was testing my Stage database data cloned into our dev region to see how the migrations of it would go, seems the certificates resource which has certs with newlines in the db had whatever migrations logic ran against it strip out all the ```\``` of the ```\n```'s causing further processing of the certs to be invalid later and messing up any kinda API backend validating Kong with MTLS due to these invalid client certs.

Pre migration public cert resource data(Kong 2.1.4), can see the newlines distinctly in the db:

![image](https://user-images.githubusercontent.com/31913027/112096577-f05c9a80-8b74-11eb-83e5-28685a990724.png)

Post migration public cert resource data(Kong 2.3.3):

![image](https://user-images.githubusercontent.com/31913027/112096657-184bfe00-8b75-11eb-961e-ce9dcc16bdcd.png)

You can see how the slashes have been dropped and n chars are left. The same holds true for the private key field as well but for obvious reasons left that out of my screenshot.

Ideally Kong does not alter these things, was the parsing/handling changed somehow or new kinda sanitizing applied to the fields.. hmm.

Oddly I think the 2.1.4 to 2.3.2 migration I practiced on our dev database records did not show this behavior. It seems the newlines were respected but due to the various ways I tested, and the original state of the dev certs unchecked before migration I can't say this with 100% degree of certainty. Just was calling it out as something I checked about our upgraded dev table and still seeing newlines in place.

Noting same would happen with certs stored with windows newline formats like ```\r\n```  , or ```\r``` in old mac os before X.  So it would be stripped and be ```rn```'s and ```r```'s

### Steps To Reproduce

1. Store your certificate resource into db w newlines and such which creates clean formatting in many UI apps like Konga UI to view your cert, Kong 2.1.4.
2. Migrate your DB to Kong 2.3.3 
3. Check DB content of your certificate resources. 
4. They been bamboozled.

### Additional Details & Logs

- Kong version 2.3.3

**Comment by bungle** [comment]
@jeremyjpj0916 we don't have any migrations that deal with certificates between those versions. Are you sure it is not somekind of client issue? Is this `cql` output?

**Comment by bungle** [comment]
we only do:
```
      ALTER TABLE certificates ADD cert_alt TEXT;
      ALTER TABLE certificates ADD key_alt TEXT;
 ```
 
 And if by that the data is changed, I would think it is a Cassandra issue.

**Comment by bungle** [comment]
I tried this by:
1. loaded `2.1.4`
2. uploaded cert with new lines
3. checked db that it contains new lines
4. switched to `2.3.3`
5. ran migrations up
6. ran migrations finish
7. checked db with cqlsh
8. checked response of certificates endpoint

So with this try, I could not reproduce it.

**Comment by jeremyjpj0916** [comment]
@bungle I had time to come back to this.

I found the issue, seems our ansible scripting that clones keyspace data around from various environments to other keyspaces/cassandra clusters is actually doing the slash stripping and causing the issue. 

So good news is its not Kong, sorry I got tied up and didn't get time to investigate this further to recognize its not part of the migrations. Closing this issue!
